Storage/Storage/Model/WooCommerce.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>Model 115.xcdatamodel</string>
+	<string>Model 116.xcdatamodel</string>
```

Comment:
I noticed you didn't added this new model under `MIGRATIONS.md`, which is important to keep up to date 👍

Reply:
I noticed that it's also missing in the testing method `test_it_can_load_the_expected_model_versions`

Reply:
🤦 I added those in https://github.com/woocommerce/woocommerce-ios/pull/13807/commits/3fda68f1f1203b3f3bf79605b4e84b70b68dab5c and https://github.com/woocommerce/woocommerce-ios/pull/13807/commits/13c16d96184a1038ccc6dfb5af54f7383a376e5b.